StellarApp/Controllers/NewsViewController.swift

```diff
@@ -1,30 +1,72 @@
-//
-//  NewsViewController.swift
-//  StellarApp
-//
-//  Created by TingxinLi on 4/8/19.
-//  Copyright © 2019 TingxinLi. All rights reserved.
-//
-
 import UIKit
 
 class NewsViewController: UIViewController {
 
+    private var articles = [ArticleWrapper]() {
+        didSet {
+            DispatchQueue.main.async {
+                self.newsView.NewsCollectionView.reloadData()
+            }
+        }
+    }
+    
+    
+    let newsView = NewsView()
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        view.addSubview(newsView)
+        let firstColor = UIColor.init(red: 255/255, green: 0/255, blue: 204/255, alpha: 1)
+        let secondColor = UIColor.init(red: 51/255, green: 51/255, blue: 153/255, alpha: 1)
+        let gradient = CAGradientLayer()
+        gradient.frame = self.newsView.NewsCollectionView.bounds
+        gradient.colors = [firstColor.cgColor, secondColor.cgColor]
```

Comment:
this is a mashup of objectives. 
```swift
addGradient()
setUpNewsCollectionView()
getArticles()
```

try to stick to the [Single Responsibility Principle](https://codeburst.io/understanding-solid-principles-single-responsibility-b7c7ec0bf80)